src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -60,9 +61,16 @@ public void execute_invalidCommandFormat_throwsParseException() {
     }
 
     @Test
-    public void execute_commandExecutionError_throwsCommandException() {
+    public void execute_deleteCommandRequiresConfirmation_displaysDeleteConfirmationMessage()
+            throws CommandException, ParseException {
         String deleteCommand = "delete 9";
-        assertCommandException(deleteCommand, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
+        assertCommandSuccess(deleteCommand, MESSAGE_DELETE_CONFIRMATION, model);
+    }
+
+    @Test
+    public void execute_commandExecutionError_throwsCommandException() throws CommandException, ParseException {
+        String[] deleteCommand = {"delete 9", "y"};
+        assertDeleteCommandFailure(CommandException.class, MESSAGE_INVALID_PERSON_DISPLAYED_INDEX, deleteCommand);
```

Comment:
nice work with these tests, but perhaps you can try asserting that if a non `y` or `yes` value is provided, then the cancellation message is shown?

Reply:
Good idea.